src/phoenix/session/client.py

```diff
@@ -450,6 +450,75 @@ def create_examples(
                 corresponding to an example in the dataset.
             dataset_description: (Optional[str]): Description of the dataset.
 
+        Returns:
+            A Dataset object with the uploaded examples.
+        """
+        return self._upload_json_dataset(
+            dataset_name=dataset_name,
+            inputs=inputs,
+            outputs=outputs,
+            metadata=metadata,
+            dataset_description=dataset_description,
+        )
+
+    def append_examples(
+        self,
+        *,
+        dataset_name: str,
+        inputs: Iterable[Mapping[str, Any]],
+        outputs: Iterable[Mapping[str, Any]] = (),
+        metadata: Iterable[Mapping[str, Any]] = (),
```

Comment:
I guess is there a way we can converge on a single API for appending examples (both table and iterables?)

Just trying to see if we can converge on a smaller API footprint but if this makes the most sense to you I get it. 

Just guess now we have a gap on the tabular "append" or?